sdk/servicebus/azure-messaging-servicebus/docs/SyncReceiveAndPrefetch.md

```diff
@@ -0,0 +1,85 @@
+### The Sync receiveMessages API and implicit Prefetch:
+
+The documentation [here][prefetchgeneral] describes the Service Bus SDK prefetch feature in general. It also states that setting the prefetch-count property in the builder to zero disables prefetch.
```

Comment:
We could add a link to this doc in azure-messaging-servicebus/README.md under the troubleshooting guide.

Reply:
Good idea! I'll do that after merging this pr to ensure the referenced file is in place.

Reply:
addressed in the pr: https://github.com/Azure/azure-sdk-for-java/pull/25885